<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaCachaza</class>
 <widget class="QMainWindow" name="ventanaCachaza">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(
        spread:pad, x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(255, 220, 150, 255), /* Color claro, hacia la izquierda */
        stop:1 rgba(230, 134, 60, 255)  /* Color naranja oscuro, hacia la derecha */
    );

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cTitulo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>900</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-family: &quot;Calibri&quot;;         /* Fuente Calibri */
    font-size: 18pt;                /* Tamaño de 20pt */
    color: #AC1F01;                 /* Color rojo oscuro especificado */
    font-weight: bold;            /* Normal, ajusta a bold si necesitas */
    text-align: left;               /* Alineación izquierda, cámbialo a center si es necesario */
	background: none;

}
</string>
    </property>
    <property name="text">
     <string>Opciones de Cachaza disponibles… Seleccione su favorito</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cBatidaCitrus">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>430</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasCachaza/batidaCitrus.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cCaipifruta">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasCachaza/caipifruta.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cCaipirinhaCoca">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>430</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasCachaza/caipirinhaCoca.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cCaipirinha">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasCachaza/caipirinha.png);
background: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cCachazaTonic">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>140</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasCachaza/cachazaTonic.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cTropicalCaipirinha">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasCachaza/tropicalCaipirinha.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cBoton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>900</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e67e22;
    color: white;                /* Texto blanco */
    font-family: Calibri;         /* Fuente Calibri */
    font-size: 22px;              /* Tamaño de la fuente */
    font-weight: bold;  
    border-radius: 20px;          /* Bordes redondeados */
    padding: 10px 20px;           /* Espaciado interno para agrandar el botón */
}

QPushButton:hover {
   background-color: #d35400;     /* Color de fondo al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cCachazaCooler">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>720</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bebidasCachaza/cachazaCooler.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>cCaipirinha</zorder>
   <zorder>cTitulo</zorder>
   <zorder>cBatidaCitrus</zorder>
   <zorder>cCaipifruta</zorder>
   <zorder>cCaipirinhaCoca</zorder>
   <zorder>cCachazaTonic</zorder>
   <zorder>cTropicalCaipirinha</zorder>
   <zorder>cBoton</zorder>
   <zorder>cCachazaCooler</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../Bebidas/bebidasCachaza/bebidasCachaza.qrc"/>
 </resources>
 <connections/>
</ui>
